test/unit/object.spec.js

```diff
@@ -82,12 +82,12 @@ describe('marionette object', function() {
       expect(this.object.destroy).to.have.returned(this.object);
     });
 
-    it('should pass an empty object options down to the onBeforeDestroy method', function() {
-      expect(this.beforeDestroyHandler).to.have.been.calledWith({});
+    it('should pass an empty array of options options down to the onBeforeDestroy method', function() {
+      expect(this.beforeDestroyHandler).to.have.been.calledWith();
     });
 
-    it('should pass an empty object options down to the onDestroy method', function() {
-      expect(this.onDestroyHandler).to.have.been.calledWith({});
+    it('should pass an empty array of options down to the onDestroy method', function() {
+      expect(this.onDestroyHandler).to.have.been.calledWith();
```

Comment:
I think the semantics of `triggerMethod('foo')` are that the handler is called w/ no params. In this one case we were calling it w/ an empty obj.